src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
While defining a config value in clrconfigvalues.h, one can also specify additional options with it to indicate that it should come only from env/registry and not from an app config file or can come from env/registry/app config file.  How is this achieved with the new mechanism?